### PR TITLE
code normalization and fix PSR-2

### PR DESCRIPTION
**Pull request by lgcosta**

### CODE REVIEW

File: live-top10-cpu-usage/src/top10_cpu.css [hunk 1]
```diff
@@ -6,28 +6,28 @@
   text-align: center;
   border: 1px grey solid;
   border-radius: 4px;
-  } 
+  }
   .bar .progress-ok {
-  background-color: #caef74;                                                                                                                                                            
+  background-color: #caef74;
```
Comment (sc979) [inline live-top10-cpu-usage/src/top10_cpu.css]: wrong indentation

File: live-top10-cpu-usage/src/table_top10cpu.ihtml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 	<link href="../../Themes/Centreon-2/Color/blue_css.php" rel="stylesheet" text="text/css"/>
 	{literal}
 	<style type="text/css">
-	  .ListTable {font-size:11px;border-color: #BFD0E2;}                                                                                                                                     
+	  .ListTable {font-size:11px;border-color: #BFD0E2;}
```
Comment (sc979) [inline live-top10-cpu-usage/src/table_top10cpu.ihtml]: wrong indentation

**Comment by lgcosta** [comment]
@sc979 done ! thankx!

**Comment by sc979** [comment]
PR cherry-picked and adapted to the 18.10.x and 19.04.x branches

**Pull request merged**